share/instruments/orders.xml

```diff
@@ -278,6 +280,8 @@
             <family>cornets</family>
             <family>flugelhorns</family>
             <family>horns</family>
+            <family>baritone-horns</family>
+            <family>euphoniums</family>
```

Comment:
I'm slightly surprised by the fact baritones/euphoniums are here **above** the trombones while in all other orders they are **below** the trombones. Is this order correct?

Reply:
That is correct for brass bands, yup.